stacks-signer/release-process.md

```diff
@@ -11,75 +11,79 @@
 | Linux ARMv7                 | _builds are provided but not tested_ |
 | Linux ARM64                 | _builds are provided but not tested_ |
 
-
 ## Release Schedule and Hotfixes
 
-Normal releases in this repository that add new or updated features shall be released in an ad-hoc manner. The currently staged changes for such releases
-are in the [develop branch](https://github.com/stacks-network/stacks-core/tree/develop). It is generally safe to run a `stacks-signer` from that branch, though it has received less rigorous testing than release branches. If bugs are found in the `develop` branch, please do [report them as issues](https://github.com/stacks-network/stacks-core/issues) on this repository.
+`stack-signer` releases that add new or updated features shall be released in an ad-hoc manner.
+It is generally safe to run a `stacks-signer` from that branch, though it has received less rigorous testing than release branches.
+If bugs are found in the `develop` branch, please do [report them as issues](https://github.com/stacks-network/stacks-core/issues) on this repository.
 
 For fixes that impact the correct functioning or liveness of the signer, _hotfixes_ may be issued. These hotfixes are categorized by priority
 according to the following rubric:
 
-- **High Priority**. Any fix for an issue that could deny service to the network as a whole, e.g., an issue where a particular kind of invalid transaction would cause nodes to stop processing requests or shut down unintentionally. 
-- **Medium Priority**. ny fix for an issue that could deny service to individual nodes.
-- **Low Priority**. Any fix for an issue that is not high or medium priority. 
+- **High Priority**. Any fix for an issue that could deny service to the network as a whole, e.g., an issue where a particular kind of invalid transaction would cause nodes to stop processing requests or shut down unintentionally.
+- **Medium Priority**. Any fix for an issue that could deny service to individual nodes.
+- **Low Priority**. Any fix for an issue that is not high or medium priority.
 
 ## Versioning
 
-This project uses a 6 part version number. When there is a stacks-core release, `stacks-signer` will assume the same version as the tagged `stacks-core` release (5 part version). When there are changes in-between stacks-core releases, the signer binary will assume a 6 part version. 
+This project uses a 6 part version number.
+When there is a stacks-core release, `stacks-signer` will assume the same version as the tagged `stacks-core` release ([5 part version](../docs/release-process.md#versioning)).
```

Comment:
We're not quite following this today, right? E.g. stacks-core is at 2.5.0.0.6, but signer releases are still 2.5.0.0.5.x

Reply:
correct - the reason is that the signer's code is still in develop branch, but a lot of the `stacks-node` code in develop was not ready for release. post 3.0, the above should be true.

Reply:
to close this loop - we did discuss bringing over the stacks-signer changes out of the develop branch, but in practice it would have been incredibly messy/difficult to accomplish without a lot of time for testing to ensure we pulled in all of the changes.